src/features/OrderPlacedPage/components/OrderPlacedPage.jsx

```diff
@@ -0,0 +1,23 @@
+import Footer from '../../Footer/components/Footer';
+import React, {useEffect} from 'react';
+import {Link} from 'react-router-dom';
+import {useDispatch} from 'react-redux';
+import {setCart} from '../../CartPage/slices/cartSlice';
+
+const OrderPlacedPage = () => {
+  const dispatch = useDispatch();
+
+  useEffect(()=>{
+    dispatch(setCart([]));
```

Comment:
This should not be handled in order placed page. Should have been handled before redirecting to this page